pkgs/development/libraries/LAStools/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub, cmake }:
+stdenv.mkDerivation rec {
+
+  version = "201003";
+  name = "LAStools-${version}";
+
+  src = fetchFromGitHub {
+    owner = "LAStools";
+    repo = "LAStools";
+    rev = "635b76b42cc4912762da31b92f875df5310e1714";
+    sha256 = "0682ca3bp51lmfp46vsjnd1bqpn05g95pf4kclvjv1y8qivkxsaq";
+  };
+
+  patches = [
+    ./drop-64-suffix.patch
+  ];
+
+  hardeningDisable = [
+    "format"
+  ];
+
+  nativeBuildInputs = [
+    cmake
+  ];
+
+  meta = {
+    description = "Software for rapid LiDAR processing";
+    homepage = http://lastools.org/;
+    license = stdenv.lib.licenses.unfree;
+    maintainers = [ stdenv.lib.maintainers.stephenwithph ];
+    platforms = stdenv.lib.platforms.unix;
+  };
```

Comment:
```suggestion
  meta = with stdenv.lib; {
    description = "Software for rapid LiDAR processing";
    homepage = http://lastools.org/;
    license = stdenv.lib.licenses.unfree;
    maintainers = with maintainers; [ stephenwithph ];
    platforms = platforms.unix;
  };
```

Reply:
fixed in 98678aff74380901114a6769adc2b161426461dd